site/src/components/gradient-card.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Gradient card
+description: The gradient card component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - gradient-card
+---
+
+# Gradient card
+
+The `.gradient-card` class contains a predefined gradient border, default border width and default border radius. The inner `.gradient-card-content` element has a default background color and padding but can be modified by Atomic classes.
+
+```html
+<div class="gradient-card tag border-radius-rounded">
+	<div class="gradient-card-content">
+		<p>Gradient card default</p>
+	</div>
+</div>
+```
+
+```html
+<div class="gradient-card">
+	<div class="gradient-card-content background-color-body-medium padding-md">
+		<p>Gradient card with atomics</p>
+	</div>
+</div>
+```
+
+## Usage
+
+Gradient cards are used to bring attention to a section or for notification messages. Here is a common use case.
```

Comment:
Honestly, that sounds misleading to me. Component itself should be a complete thing, not a "supportive" wrapper for something else.

Also, by reading this I'd expect to see an example like this, but I don't believe that's the purpose of the `gradient-card` component:

```html
<div class="gradient-card">
	<div class="gradient-card-content">
		<div class="notification">
  			<p>A notification about something happening in the world <a href="#">Link</a></p>
		</div>
	</div>
</div>
```

Reply:
![image](https://github.com/microsoft/atlas-design/assets/57374379/f2beb2d7-55e7-4edf-b540-1a184ec12d72)

Reply:
> @homing1 @ollips
> 
> This is for the AI Copilot branding, it is designed to be used in a couple scenarios. For larger blocks of content like a card: <img alt="image" width="518" src="https://user-images.githubusercontent.com/38262647/238470893-e40c6024-c418-48c6-9533-2d6238e28bf7.png">
> 
> And also for notifications, usually paired with the sparkle docon icon. <img alt="image" width="612" src="https://user-images.githubusercontent.com/38262647/238471211-6f43b159-ce54-45bd-86dc-8de260ee96f7.png">

From what I see on these screenshots - this component won't be mixed with the `notification` component, yay, so please update the wording so no one is confused by it.

Reply:
Makes sense, updated the messaging to remove any 'banner' or 'notification' references.